msg/messages.js

```diff
@@ -261,59 +263,70 @@ Blockly.Msg.SOUND_CHANGEVOLUMEBY = "change volume by %1";
+
 // Context menus
-Blockly.Msg.DUPLICATE = 'Duplicate';
-Blockly.Msg.DELETE = 'Delete';
-Blockly.Msg.ADD_COMMENT = 'Add Comment';
-Blockly.Msg.REMOVE_COMMENT = 'Remove Comment';
-Blockly.Msg.DELETE_BLOCK = 'Delete Block';
-Blockly.Msg.DELETE_X_BLOCKS = 'Delete %1 Blocks';
-Blockly.Msg.DELETE_ALL_BLOCKS = 'Delete all %1 blocks?';
-Blockly.Msg.CLEAN_UP = 'Clean up Blocks';
-Blockly.Msg.HELP = 'Help';
-Blockly.Msg.UNDO = 'Undo';
-Blockly.Msg.REDO = 'Redo';
-Blockly.Msg.EDIT_PROCEDURE = 'Edit';
-Blockly.Msg.SHOW_PROCEDURE_DEFINITION = 'Go to definition';
-Blockly.Msg.WORKSPACE_COMMENT_DEFAULT_TEXT = 'Say something...';
+Blockly.Msg.DUPLICATE = "Duplicate";
+Blockly.Msg.DELETE = "Delete";
+Blockly.Msg.ADD_COMMENT = "Add Comment";
+Blockly.Msg.REMOVE_COMMENT = "Remove Comment";
+Blockly.Msg.DELETE_BLOCK = "Delete Block";
+Blockly.Msg.DELETE_X_BLOCKS = "Delete %1 Blocks";
+Blockly.Msg.DELETE_ALL_BLOCKS = "Delete all %1 blocks?";
+Blockly.Msg.CLEAN_UP = "Clean up Blocks";
+Blockly.Msg.HELP = "Help";
+Blockly.Msg.UNDO = "Undo";
+Blockly.Msg.REDO = "Redo";
+Blockly.Msg.EDIT_PROCEDURE = "Edit";
+Blockly.Msg.SHOW_PROCEDURE_DEFINITION = "Go to definition";
+Blockly.Msg.WORKSPACE_COMMENT_DEFAULT_TEXT = "Say something...";
 
 // Color
-Blockly.Msg.COLOUR_HUE_LABEL = 'Color';
-Blockly.Msg.COLOUR_SATURATION_LABEL = 'Saturation';
-Blockly.Msg.COLOUR_BRIGHTNESS_LABEL = 'Brightness';
+Blockly.Msg.COLOUR_HUE_LABEL = "Color";
+Blockly.Msg.COLOUR_SATURATION_LABEL = "Saturation";
+Blockly.Msg.COLOUR_BRIGHTNESS_LABEL = "Brightness";
 
 // Variables
 // @todo Remove these once fully managed by Scratch VM / Scratch GUI
-Blockly.Msg.CHANGE_VALUE_TITLE = 'Change value:';
-Blockly.Msg.RENAME_VARIABLE = 'Rename variable';
-Blockly.Msg.RENAME_VARIABLE_TITLE = 'Rename all "%1" variables to:';
-Blockly.Msg.RENAME_VARIABLE_MODAL_TITLE = 'Rename Variable';
-Blockly.Msg.NEW_VARIABLE = 'Make a Variable';
-Blockly.Msg.NEW_VARIABLE_TITLE = 'New variable name:';
-Blockly.Msg.VARIABLE_MODAL_TITLE = 'New Variable';
-Blockly.Msg.VARIABLE_ALREADY_EXISTS = 'A variable named "%1" already exists.';
-Blockly.Msg.VARIABLE_ALREADY_EXISTS_FOR_ANOTHER_TYPE = 'A variable named "%1" already exists for another variable of type "%2".';
-Blockly.Msg.DELETE_VARIABLE_CONFIRMATION = 'Delete %1 uses of the "%2" variable?';
-Blockly.Msg.CANNOT_DELETE_VARIABLE_PROCEDURE = 'Can\'t delete the variable "%1" because it\'s part of the definition of the function "%2"';
-Blockly.Msg.DELETE_VARIABLE = 'Delete the "%1" variable';
+Blockly.Msg.CHANGE_VALUE_TITLE = "Change value:";
+Blockly.Msg.RENAME_VARIABLE = "Rename variable";
+Blockly.Msg.RENAME_VARIABLE_TITLE = "Rename all '%1' variables to:";
+Blockly.Msg.RENAME_VARIABLE_MODAL_TITLE = "Rename Variable";
+Blockly.Msg.NEW_VARIABLE = "Make a Variable";
+Blockly.Msg.NEW_VARIABLE_TITLE = "New variable name:";
+Blockly.Msg.VARIABLE_MODAL_TITLE = "New Variable";
+Blockly.Msg.VARIABLE_ALREADY_EXISTS = "A variable named '%1' already exists.";
```

Comment:
Are we okay with this user visible display message quoting the selected variable name changing to use single quotes? E.g. in this message:

![image](https://user-images.githubusercontent.com/1786240/41314445-c2e2f7c8-6e5a-11e8-93db-97a98dfa9512.png)

Reply:
Good catch.  Similar quote characters are changed below.

Reply:
I think it was originally because we're generating json, and that requires double quotes. Given that we mix apostrophe and double quotes in the same string, we're going to be escaping things one way or the other. I'm changing messages.js to require single quotes, but really it could go either way.